src/configuration/configuration.ts

```diff
@@ -307,6 +307,27 @@ class ConfigurationClass {
    */
   foldfix = false;
 
+  /**
+   * Determines whether VSCodeVim is in Disabled mode or not.
+   */
+  private _disableExtension: boolean | undefined;
```

Comment:
Does it make sense to set this to `false` to start? Is there a scenario where it'll be `undefined`?

Reply:
I have some weird behavior I need to work out, and this will be involved in some further changes I need to make. I'll let you know when it's ready. Sorry about all the confusion / edits!